libvcx/src/aries/handlers/proof_presentation/prover/state_machine.rs

```diff
@@ -199,23 +192,30 @@ impl ProverSM {
         Ok(ProverSM { source_id, state, thread_id })
     }
 
-    fn _handle_reject_presentation_request(connection_handle: u32, reason: &str, presentation_request: &PresentationRequest, thread_id: &str) -> VcxResult<()> {
+    fn _handle_reject_presentation_request(
```

Comment:
I don't think the `_` prefix is necessary in Rust

Reply:
Though it's not official Rust convetion, it is convention all across the repo (try search for `fn _`). I find it personally useful

Reply:
OK, let's keep it that way